opsdroid/core.py

```diff
@@ -485,6 +486,8 @@ async def get_ranked_skills(self, skills, message):
                 _LOGGER.debug(_("Checking Rasa NLU..."))
                 ranked_skills += await parse_rasanlu(self, skills, message, rasanlu)
 
+        if isinstance(message, events.Message) and not ranked_skills:
```

Comment:
This question is one for discussion, I am not really sure one way or another, but do we want this "catch all" to catch things that aren't messages? Should it catch all unhandled events, i.e. joins, images, files etc. My feeling is that if it's going to be named `match_catchall` then it probably should, if it stays as messages only it should be renamed to reflect that?

Reply:
I think we should have both. A catch all and a catch all messages.

Reply:
So I will add a `messages_only` flag to decorator, defaulting to False?